src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -154,24 +156,63 @@ public AzureAppConfigurationOptions UseFeatureFlags(Action<FeatureFlagOptions> c
                     string.Format(ErrorMessages.CacheExpirationTimeTooShort, MinimumFeatureFlagsCacheExpirationInterval.TotalMilliseconds));
             }
 
-            if (!(_kvSelectors.Any(selector => selector.KeyFilter.StartsWith(FeatureManagementConstants.FeatureFlagMarker) && selector.LabelFilter.Equals(options.Label))))
+            if (options.FeatureFlagSelectors.Count() != 0 && options.Label != null)
             {
```

Comment:
The previous approach doesn't pull any additional feature flags if the user selected feature flags already. Is it intentional to shift away from that approach?

Reply:
We will still check that before registering new kv selectors [here](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/234/files/e4b6e329af81c52ec6f0f9195f39a6ddbad3dbcf#diff-9fe8bf4d21db7a9a52895366df1e58edce9f4d7dca80514f7b457311f3218cecR195). That behavior has not changed.

Reply:
Probably will have to get on a call about this one. I may need some help :)